log/file.go

```diff
@@ -135,6 +139,7 @@ func (h *fileHook) loop(ctx context.Context) chan []byte {
 					h.fallbackLogger.Errorf("failed to close logfile: %w", err)
 				}
 
+				close(h.done)
```

Comment:
I think this should be in a defer, WDYT?

Reply:
resolved with the latest force push